### PR TITLE
Fix android manifest file provider for Android X migration

### PR DESCRIPTION
**Pull request by davidlutta**
For users who have updated their projects to android X this library does not seem to work

### CODE REVIEW

**Comment by jhansireddy** [comment]
Could you send all the file changes needed for android x migration.

File: ScanDemoExample/scanlibrary/src/main/java/com/scanlibrary/PickImageFragment.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.provider.MediaStore;
-import android.support.v4.content.FileProvider;
+import androidx.core.content.FileProvider;
```
Comment (Priyanshu-CODERX) [inline ScanDemoExample/scanlibrary/src/main/java/com/scanlibrary/PickImageFragment.java:13]: Actually I tried to do this but while saving the image its crashing

Comment (vamsee9) [reply ScanDemoExample/scanlibrary/src/main/java/com/scanlibrary/PickImageFragment.java:13]: for me after clicking pic it shows black screen with folder and camera option it's not directing to crop and filters

**Comment by jhansireddy** [comment]
Have these changes already incorporated into the codebase. Hence closing the PR.

**Pull request closed without merging**